README.md

```diff
@@ -140,19 +140,17 @@ For a cluster with 2 worker pools, named 'default' and 'secondarypool', follow t
       $ terraform state list | grep ibm_container_vpc_worker_pool
         > module.ocp_base.data.ibm_container_vpc_worker_pool.all_pools["default"]
         > module.ocp_base.data.ibm_container_vpc_worker_pool.all_pools["secondarypool"]
+        > module.ocp_base.ibm_container_vpc_worker_pool.pool["default"]
+        > module.ocp_base.ibm_container_vpc_worker_pool.pool["secondarypool"]
         > ...
 
-      $ terraform state rm "module.ocp_base.ibm_container_vpc_worker_pool.all_pools[\"default\"]"
-      $ terraform state rm "module.ocp_base.ibm_container_vpc_worker_pool.all_pools[\"secondarypool\"]"
-      $ ...
+      $ terraform state rm "module.ocp_base.ibm_container_vpc_worker_pool.pool[\"default\"]"
 ```
 
 Schematics Example: For a cluster with 2 worker pools, named 'default' and 'secondarypool', follow these steps:
```

Comment:
Do we even need to mention 'secondarypool' here? Regardless of how many other pools there are or what they are called, your always going to run the state remove command on the default pool right?

Reply:
Yes